src/user/userService.ts

```diff
@@ -1,89 +0,0 @@
-import bcrypt from 'bcryptjs';
-import { BadRequest, UnprocessableEntity, Unauthorized } from 'http-errors';
-import { UserInput } from './interfaces';
-import { User } from '../libs/db/schema';
-import UserRepository from './userRepository';
-import TokenService from '../libs/common/tokenService';
-import EmailService from './emailService';
-
-class UserService {
-    constructor(
-        private userRepository = UserRepository,
-        private tokenService = TokenService,
-        private emailService = EmailService,
-    ) { }
-
-    async registerUser(input: UserInput): Promise<User> {
-        const { email, password } = input;
-
-        if (!email || !password) {
-            throw new BadRequest('Please provide email and password');
-        }
-
-        const userEmail = email.trim();
-        const userPassword = password.trim();
```

Comment:
What if the password has a space!!

Reply:
@OmarMuhtaseb i was under assumption spaces shouldn't be allowed in passwords. i was wrong lol